tests/php/Extensions/ContentReviewCMSExtensionTest.php

```diff
@@ -53,7 +54,7 @@ public function testGetReviewContentFormThrowsExceptionWhenObjectCannotBeReviewe
         $mock->setOwner(new Controller);
 
         // Return a DataObject without the content review extension applied
-        $mock->expects($this->once())->method('findRecord')->with(['ID' => 123])->willReturn(new Member);
+        $mock->expects($this->once())->method('findRecord')->with(['ID' => 123])->willReturn(new SiteTree);
```

Comment:
Why is the return type changing here?

Reply:
The return type for `ContentReviewCMSExtension::findRecord` is `SiteTree`
https://github.com/silverstripe/silverstripe-contentreview/blob/4/src/Extensions/ContentReviewCMSExtension.php#L107

Reply:
Does this mean the test didn't work before (and may still not work now)?